person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_reader :parent_permission
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
```

Comment:
- [ ] Kindly initialize `id` here as well, and the `id` should be randomly generated using the random function as this is a requirement.
e.g `@id = Random.rand(1..1000)`